ci/test_onnx2ncnn.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
We've already made a folder '.github/scripts'. Let's move `test_onnx2ncnn.py` to it.

Reply:
done.